contents/blog/posthog-vs-pendo.md

```diff
@@ -216,7 +216,7 @@ This difference is reflected in all levels of the product, but especially in pro
 
 - **SQL access:** While both PostHog and Pendo have ready-made insights and visualization types, only PostHog gives you unlimited access to your data by [writing your own SQL queries](/docs/product-analytics/hogql). This is ideal for data scientists, product managers, and engineers who want to perform advanced analysis on user data.
 
-<h4 className="mb-0">Discover what's possible with <span className="text-blue">product analytics</span></h4>
+<h4 className="mb-4">Discover what's possible with <span className="text-blue">product analytics</span></h4>
```

Comment:
Before and after:

<img width="629" alt="Screenshot 2023-05-10 at 1 46 55 PM" src="https://github.com/PostHog/posthog.com/assets/7090054/a6cae059-7bec-4222-a2e7-e002f336f64e">
<img width="834" alt="Screenshot 2023-05-10 at 6 00 36 PM" src="https://github.com/PostHog/posthog.com/assets/7090054/fcfa7a8a-ba08-4c15-851c-b4f358704846">